src/main/scala/com/github/unchama/seichiassist/menus/minestack/MineStackSelectItemColorMenu.scala

```diff
@@ -21,7 +21,7 @@ object MineStackSelectItemColorMenu {
 
 }
 
-case class MineStackSelectItemColorMenu(mineStackObj: MineStackObj) extends Menu {
+case class MineStackSelectItemColorMenu(mineStackObj: MineStackObject) extends Menu {
```

Comment:
ここで
```suggestion
case class MineStackSelectItemColorMenu(group: GroupedMineStackObjects) extends Menu {
```
を受け取ってしまえば、 37行目から46行目までの検索はそもそも行う必要が無い。